### ISSUE TITLE
Ensure NGINX compatibilty?

### ISSUE THREAD

**Issue by M4LuZ** [lansuite/lansuite#889]
## Expected Behavior

We use NGINX in the docker setup as webserver and thus one can expect it to be properly supported.
We ensure some expected configuration for Apache via `.htaccess`

## Current Behavior

We do not do the same for NGINX-Installations, so it is up to the user to ensure correct configuration

## Possible Solution

Provide NGINX config either in documentation or dedicated config file. 
Tbh I have only experience with Apache, so not sure what options there are.
It is clear at least that something else than `.htaccess` is needed: https://www.nginx.com/resources/wiki/start/topics/examples/likeapache-htaccess/

**Comment by andygrunwald** [comment]
Basic configuration is available at https://github.com/lansuite/lansuite/tree/master/.docker

**Comment by M4LuZ** [comment]
Yes, but that is the _basic_ configuration for the _development_ environment.
If that config is also fit for productive use, then we should mark and document it as such (at least in the installation instructions).
If it is not, then that should be provided additionally
